src/main/java/object/coordinate/Position.java

```diff
@@ -0,0 +1,92 @@
+package object.coordinate;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Position {
+    private static int BOARD_MIN_POSITION_THRESHOLD = 0;
+    private static int BOARD_ROW_POSITION_THRESHOLD = 9;
+    private static int BOARD_COLUMN_POSITION_THRESHOLD = 8;
+
+    private final int row;
+    private final int column;
+
+    public Position(int row, int column) {
+        validate(row, column);
+        this.row = row;
+        this.column = column;
+    }
+
+    private void validate(int row, int column) {
+        if (!(row >= BOARD_MIN_POSITION_THRESHOLD && row <= BOARD_ROW_POSITION_THRESHOLD)) {
+            throw new IllegalStateException("보드판 범위 밖의 이동입니다.");
+        }
+        if (!(column >= BOARD_MIN_POSITION_THRESHOLD && column <= BOARD_COLUMN_POSITION_THRESHOLD)) {
+            throw new IllegalStateException("보드판 범위 밖의 이동입니다.");
+        }
+    }
+
+    public static Position parseFrom(String rawTextCoordinate) {
+        String[] parsedText = rawTextCoordinate.split("\\s*,\\s*");
+        if (parsedText.length != 2) {
+            throw new IllegalArgumentException("올바른 좌표 입력이 아닙니다. (y, x) 형태로 입력해주세요.");
+        }
+        int row = Integer.parseInt(parsedText[0].trim());
+        int column = Integer.parseInt(parsedText[1].trim());
+
+        return new Position(row, column);
+    }
+
+    public static Position getAbsoluteBigPositionBetween(Position position, Position otherPosition) {
+        if (position.isAbsoluteBigger(otherPosition)) {
+            return position;
+        }
+        return otherPosition;
+    }
+
+    private boolean isAbsoluteBigger(Position position) {
+        return (this.row + this.column) - (position.row + position.column) > 0;
+    }
+
+    public Position add(RelativePosition relativePosition) {
+        int newRow = this.row + relativePosition.getRow();
+        int newColumn = this.column + relativePosition.getColumn();
+
+        return new Position(newRow, newColumn);
+    }
+
+    public Position add(RelativePath relativePath) {
+        Position resultPosition = this;
+        List<RelativePosition> positions = relativePath.getRelativePositions();
+        for (RelativePosition position : positions) {
+            resultPosition = resultPosition.add(position);
+        }
```

Comment:
위치에 위치를 더하는게 의미적으로 이상한 것 같습니다. 어쩌면 RelativePosition이 아니라 Direction이 맞을 것 같다는 생각이 드네요. 아래와 같은 코드는 어떤가요?

```suggestion
        for (Direction direction : relativePath) {
            resultPosition = resultPosition.apply(direction);
        }
```

Reply:
`Direction`이라는 네이밍이 확실히 좋네요. 적용하도록 하겠습니다.
저도 읽으면서 조금 이해가 안되는 느낌을 받았던 것 같은데. 좋은 피드백 감사드립니다 :)